test/unit/Fixture/PropertyHooks.php

```diff
@@ -164,3 +164,8 @@ class FinalPropertyHooks
         final set => strtolower($value);
     }
 }
+
+interface InterfaceWithProperty
+{
+    public int $abstractPropertyFromInterface { get; set; }
```

Comment:
What happens on the concrete class / implementation? Is the declarer changed / the property becomes non-abstract?

Reply:
In class the `abstract` keyword is required so we already have that covered: https://3v4l.org/KcHto#v8.4.3

Reply:
Fun! thanks for clarifying!